kustomize/overlays/dev/deployment.yaml

```diff
@@ -15,11 +15,7 @@ spec:
             - name: ENVIRONMENT
               value: dev
             - name: REPLICATED_INSTALL_URL
-              value: http://localhost:8090
-            - name: GRAPHQL_PREM_ENDPOINT
-              value: http://docker.for.mac.host.internal:8033/graphql
-            - name: REGISTRY_ENDPOINT
-              value: 'registry.staging.replicated.com' # TODO
+              value: https://emosbaugh.ngrok.io
```

Comment:
is this correct?
It feels like it was unintentionally committed

Reply:
thanks